bb8/src/inner.rs

```diff
@@ -128,6 +130,10 @@ where
         };
 
         self.inner.statistics.record(kind);
+        if let Some(wait_time_start) = wait_time_start {
+            let wait_time = Instant::now() - wait_time_start;
+            self.inner.statistics.record_get(wait_time);
+        }
```

Comment:
Please look again at the feedback I left in the previous PR.

Reply:
It took me 2 hours to see this `Comment on lines +135 to +142` ... shit

Reply:
Yes I was missing this "little" detail